### PR TITLE
(libreoffice*) Change package name to follow upstream rules

### PR DESCRIPTION
**Pull request by wget**
Fix [#458](https://github.com/chocolatey/chocolatey-coreteampackages/issues/458)

### CODE REVIEW

File: deprecated/packages/libreoffice-oldstable/libreoffice-oldstable.nuspec [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    <licenseUrl>https://www.libreoffice.org/download/license/</licenseUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <iconUrl></iconUrl>
+    <releaseNotes>https://www.libreoffice.org/download/release-notes</releaseNotes>
+    <packageSourceUrl>https://github.com/chocolatey/chocolatey-coreteampackages/tree/master/automatic/libreoffice-oldstable</packageSourceUrl>
+    <projectSourceUrl>https://www.libreoffice.org/about-us/source-code</projectSourceUrl>
+    <docsUrl>https://www.libreoffice.org/get-help/documentation</docsUrl>
+  </metadata>
+  <dependencies>
+    <dependency id="libreoffice-fresh" version="5.1.6" />
```
Comment (AdmiringWorm) [inline deprecated/packages/libreoffice-oldstable/libreoffice-oldstable.nuspec]: wrong dependency, should be libreoffice-still

**Comment by wget** [comment]
@AdmiringWorm Yep. I saw that it was still using ketarin and I also saw the package was not providing a 64 bits version while it should.

**Comment by AdmiringWorm** [comment]
@wget that's not all, it also do not provide checksums.
I believe I have updating the libreoffice-(fresh)-help package on my todo list though.
Just never got around to it yet, this is also why I suggested that perhaps remove that package from this PR.

**Comment by AdmiringWorm** [comment]
@wget could you also rebase your changes please?

**Comment by AdmiringWorm** [comment]
@wget what's the plan regarding libreoffice-help?
I can't really approve this PR yet, until the changes for that package is either removed, or converted to AU.

**Comment by wget** [comment]
@AdmiringWorm I'll take care of it as soon I have some free time. The help package is not an easy task especially as I need to detect the exact libreoffice version installed, bringing huge changes to this package etc. I thought I would have been able to make it today, but I'm now debugging some appveyor build script, so this topic will wait a few more days :-)

**Comment by AdmiringWorm** [comment]
> as I need to detect the exact libreoffice version installed

I don't really think that would be appropriate.
Since the libreoffice-help packages should be versioned the same as the main packages, it should be up to the user to specify the version used (if he/she uses an older version of libreoffice).

/cc @wget

**Comment by wget** [comment]
What I meant by `the libreoffice version installed` is whether the user is using the 32 or the 64 bits version. LibreOffice now provides 2 packages for the help files and their location depends on the libreoffice version used. Obviously the 32 and 64 bits versions are not located in the same folder. Nothing can prevent an user to install a 32 bits of LibreOffice on a 64 bits Windows installation.

**Comment by AdmiringWorm** [comment]
@wget what's the status on this one?
Any change?
I don't want to see this PR closed, but it will if no progress is being made.
Alternatively if you are unable to continue working on the `libreoffice-help*` packages, then you could remove them and we can to a new complete review of the other packages (Guidelines have changed since this PR was initially opened)

**Comment by majkinetor** [comment]
Closed as there is no feedback

**Pull request closed without merging**